core/src/main/java/hera/core/commands/Bind.java

```diff
@@ -0,0 +1,62 @@
+package hera.core.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import discord4j.core.object.entity.*;
+import hera.core.HeraUtil;
+import hera.core.messages.MessageHandler;
+import hera.core.messages.MessageSpec;
+import hera.database.entities.Binding;
+import hera.database.entities.BindingType;
+import hera.database.types.SnowflakeType;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+import static hera.store.DataStore.STORE;
+
+public class Bind {
+	public static Mono<Void> execute(MessageCreateEvent event, Guild guild, Member member, MessageChannel channel, List<String> params) {
+		try {
+			List<BindingType> bTypes = STORE.bindingTypes().forName(params.get(0).trim().toUpperCase());
+			if (!bTypes.isEmpty()) {
+				BindingType bindingType = bTypes.get(0);
+				switch (bindingType.getSnowflakeType()) {
+					case ROLE:
+						Mono<Role> role = HeraUtil.getRoleFromMention(guild, params.get(1));
```

Comment:
We should send an error message when the role which the user passes in is not an actual role. Right now this would fail silently I think.

Same for the `getChannelFromMention` in the `CHANNEL` case.